.github/workflows/hera_pr.yaml

```diff
@@ -41,6 +41,93 @@ jobs:
 
       - name: Run tests
         env:
+          ENABLE_BDIST_EXT_MODULE: 'yes'
           TOXENV: python${{ matrix.python-version }}
         run: |
           tox
+
+      - name: Check build
+        run: |
+          twine check .tox/dist/*
+
+      - name: upload build artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: pypi-build-arts
+          path: .tox/dist
+
+      - name: upload coverage files
+        uses: actions/upload-artifact@v2
+        with:
+          name: coverage-data
+          path: ".tox/.coverage.*"
```

Comment:
Does the publication of coverage files allow one to construct a test coverage GitHub badge? I am a bit unfamiliar with the necessary setup of coverage so I am very excited this is added, and I would love to see the report on the main repo README! I must say a direct thank you here, as a comment, for the efforts 🚀

Reply:
With pleasure. 

Yes, we are definitely _going for a n honorable shield_. 

See `coverage` in https://github.com/argoproj-labs/hera-workflows/pull/101#issuecomment-1065169028